src/Utilities/SeekCamera/SeekCam.cpp

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
Add their license to all code files as well.

Reply:
Ok, thank you.